developer/contributing/pull-requests.rst

```diff
@@ -0,0 +1,74 @@
+Creating a Pull Request
+=======================
+
+* Use a meaningfull name for the pull request.
+* Create the pull request as soon as possible.
+* Name the branch after the following format: ``[feature|bugfix|hotfix|enhancement]/[issue-number]-<description>``
+
+ * The issue number is optional, but should always be provided if there is an issue
+
+* Add a label for the type of the PR (feature, bugfix, hotfix, enhancement)
+* Add a label for the state of the PR
+
+ * In progress if you are still working on the PR
+ * Review when it is reviewable
+ * Feedback if you have reviewed someone else's pr
+ * Alternatively you can also use the `Waffle board`_
+
+* Template is devided in two parts: tasks (for developer), informations (for reviewer)
+* Tasks list can be extended for developer tasks
+* Information: tests => if tests fail, add reason
+* Information: fixed tickets => tickets to close after merge
+* Information: bc breaks
+
+Template
+--------
+
+.. code-block:: none
+
+    [short description]:
+
+    __tasks:__
+
+    - [ ] test coverage
+    - [ ] gather feedback for my changes
+    - [ ] submit changes to the documentation
+    - [ ] ... <add your own tasks>
+
+
+    __informations:__
+
+    | q             | a
+    | ------------- | ---
+    | tests pass?   | [yes|no => <why?>]
+    | fixed tickets | [<which?>|none]
+    | bc breaks     | [description]
+    | doc           | [url to commit|none]
+
+Example
+-------
+
+Create user command:
+
+.. code-block:: none
+
+    __tasks:__
+
+    - [ ] test coverage
+    - [ ] gather feedback for my changes
+    - [ ] submit changes to the documentation
+    - [ ] remove request defaults
+    - [ ] fix #1
+
+
+    __informations:__
+
+    | q             | a
+    | ------------- | ---
+    | bug fix?      | no
+    | new feature?  | yes
+    | tests pass?   | no => __dependency__ https://github.com/sulu-cmf/sulu/pull/86
+    | fixed tickets | #1 , #2
+    | doc           | https://github.com/massiveart/sulu-docs/blob/master/developer-documentation/400-contacts/create-user.md
```

Comment:
Usually we copy the URL to the commit, so that the changes can be easier reviewed.